app/utils/__init__.py

```diff
@@ -207,3 +207,12 @@ def get_defconfig_full(
             defconfig_full = defconfig_full_k
 
     return defconfig_full
+
+
+def make_path(dir_path):
+    if not os.path.isdir(dir_path):
+        try:
+            os.makedirs(dir_path)
+        except OSError as e:
+            if e.errno != errno.EEXIST:
+                raise e
```

Comment:
Missing line return.

Reply:
@mgalka It's still there - not a big thing, but git will keep complaining about it and it's good practice to avoid it.